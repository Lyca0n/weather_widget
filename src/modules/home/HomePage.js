import React from 'react';
import Splash from '../common/Loader';
import  WeatherWidget  from './WeatherWidget';

const HomePage = () => (
    <div className="page-container">
        <div className="section" >
            <div className="content-container">
                <div className="section__title">
                    A HOME PAGE
                </div>
                <div className="cards__container">
                    <div className="card">
                        <h4>ONE</h4>
                        <WeatherWidget />

                    </div>
                    <div className="card">
                        <h4>TWO</h4>

                    </div>
                    <div className="card">
                        <h4>THREE</h4>
                    </div>
                </div>
            </div>
        </div>

    </div>
);

export default HomePage;