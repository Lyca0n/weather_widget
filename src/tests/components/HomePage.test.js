import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../../modules/home/HomePage';

test('should render DashboardPage correctly', () => {
  const wrapper = shallow(<HomePage />);
  expect(wrapper).toMatchSnapshot();
});
