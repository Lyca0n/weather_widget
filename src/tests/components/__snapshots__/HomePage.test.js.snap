// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DashboardPage correctly 1`] = `
<div
  className="page-container"
>
  <div
    className="section"
  >
    <div
      className="content-container"
    >
      <div
        className="section__title"
      >
        A HOME PAGE
      </div>
      <h4>
        Weather
      </h4>
      <Connect(WeatherWidget) />
    </div>
  </div>
</div>
`;
